frameworks/elastic/docs/limitations.md

```diff
@@ -15,13 +15,19 @@ The maximum number of deployable nodes is constrained by the DC/OS cluster's res
 
 During deployment and upgrades, the `serial` strategy does not wait for the Elastic service to reach green before proceeding to the next node.
 
+## Toggling TLS requires doing a full-cluster restart
+
+This is an [Elasticsearch limitation](https://www.elastic.co/guide/en/elasticsearch/reference/current/configuring-tls.html). Since the service's default update strategy is a rolling upgrade (`serial`) changing its configuration to enable (or disable) transport encryption will result in nodes configured with TLS not being able to communicate with nodes configured with unencrypted networking (and vice-versa). You must do a full-cluster restart (`parallel` update strategy). Make sure to have backups and plan for some downtime.
```

Comment:
Is "unencrypted networking" the standard terminology for this in Elasticsearch? In the other services we use "plaintext".

Reply:
The cluster restart sentence could be worded as:
"For this a full-cluster restart is required (enabled using the `parallel` update strategy)"

Reply:
> Is "unencrypted networking" the standard terminology for this in Elasticsearch? In the other services we use "plaintext".

Yes. I tried to use ES terminology for "node" configuration and SDK terminology for "service" configuration. I'll switch to "plaintext" so that we're consistent.

Reply:
> The cluster restart sentence could be worded as:
"For this a full-cluster restart is required (enabled using the parallel update strategy)"

In that sentence, reading through the whole paragraph, doesn't it sound like "this" is referring to "nodes configured with TLS not being able to communicate with nodes configured with unencrypted networking", which could be a bit confusing?

What do you think about "A full-cluster restart is required (using the `parallel` update strategy)"?

Reply:
Feel free to continue using the ES terminology if you think it's better.

I like the "A full-cluster restart ..." suggestion.